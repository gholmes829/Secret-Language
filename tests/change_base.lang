# convert to bin, hex, and dec

num main() {

    let val = 39
    print(bin(val))
    print(hex(val))

    return 0
}