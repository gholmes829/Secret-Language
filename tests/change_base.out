0b100111
0x27
